src/Frame.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+function Frame(id){
+  this.MAX_SCORE = 10;
+  this.id = id;
+  this.totalFrame = [];
+}
+
+Frame.prototype.inputBowls = function(first, second) {
+  var pins = first + second;
+  this.checkNumberOfPins(pins);
+  this.totalFrame.push(first, second);
+};
+
+Frame.prototype.checkNumberOfPins = function(pins) {
```

Comment:
Could the name of checkNumberOfPins be even more specific? What is it checking about the number?